data/random-teams.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+/** @typedef {import('../sim/prng').PRNG} PRNG */
+
 /** @type {typeof import('../sim/dex').Dex} */
 const Dex = require(/** @type {any} */ ('../.sim-dist/dex')).Dex;
 /** @type {typeof import('../sim/prng').PRNG} */
```

Comment:
Worth simplifying this type annotation at some point, given the typedef?

Reply:
Sadly, no, this can't be simplified.

Reply:
Maybe we could change to `.ts`, though.

Reply:
I tried that but TypeScript complained because you can't assign a type to a value.

Reply:
Oh, wait, I see what you're saying. Yes, this can be `/** @type {typeof PRNG} */`

Reply:
It's such a minor optimization, though.

Reply:
You can't do `/** @type {typeof PRNG} /*` because the variable itself is called `PRNG`.

So you need to rename to something like `PRNGType` (which is why it was called that initially that), but you'll arrive to this error:

```
data/random-teams.js:7:19 - error TS2693: 'PRNGType' only refers to a type, but is being used as a value here.
```

Reply:
Oh, yeah, that makes sense too. Oh well, it works as-is.

Reply:
Thanks for explaining.